hardware/plugins/PluginMessages.h

```diff
@@ -140,6 +142,21 @@ namespace Plugins {
 		DisconnectMessage(int HwdID) : CPluginMessage(PMT_Disconnect, HwdID) {};
 	};
 
+	class CommandMessage : public CPluginMessage
+	{
+	public:
+		CommandMessage(int HwdID, int Unit, const std::string & Message, const int level, const int hue) : CPluginMessage(PMT_Command, HwdID, Unit, Message, level, hue)
+		{
+			m_fLevel = -273.15;
```

Comment:
Ah, I see what you did, comparison to that value in both places
Here is a fix
https://github.com/domoticz/domoticz/pull/1245